### ISSUE TITLE
end to end sparse embedding benchmark (single machine, cpu)

### ISSUE THREAD

**Issue by eric-haibin-lin** [eric-haibin-lin/mxnet#85]
### model 
input (sparse format) -> matrix-vector multiplication -> linear_regression_output, MSE loss.
Benchmark script commit 2612b30

### dataset
sampled [kdda](https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/binary.html#kdda) dataset, preloaded to memory, no IO cost. Number of features = 20M. Batch size = 512

### setup
c3.8xlarge. trained with 5 epochs, 1000 batches per epoch.

### result 
| framework | version | time(s) |
|:-------:|------| ------|
| Tensorflow | tensorflow-1.1.0 | 165
| MXNet | a1597cd | 37 (4.45x speedup)

**Comment by eric-haibin-lin** [comment]
@mli
